TLM/TMPE.API/Traffic/Enums/ExtVehicleType.cs

```diff
@@ -27,15 +27,16 @@ public enum ExtVehicleType {
         PassengerFerry = 1 << 16,
         PassengerBlimp = 1 << 17,
         CargoPlane = 1 << 18,
+        Trolleybus = 1 << 20,
```

Comment:
Why 20 and not 19?

Reply:
Damn... I don't know what will break when I fix that value 😕

Reply:
Anyways, good catch, I will test what happen, but I need to create new game for that

Reply:
Does that info get stored in save?

Reply:
I must check, if not I will update it